packages/bluesnap-direct-integration/src/utils/is-bluesnap-direct-instrument.ts

```diff
@@ -29,3 +30,11 @@ export function isSepaInstrument(paymentData: unknown): paymentData is WithSepaI
             'shopperPermission' in paymentData,
     );
 }
+
+export function isPayByBankInstrument(
+    paymentData: unknown,
+): paymentData is WithPayByBankInstrument {
+    return Boolean(
+        typeof paymentData === 'object' && paymentData !== null && 'iban' in paymentData,
```

Comment:
I would add an additional check that paymentData doesn't have firstName, lastName, shopperPermission to avoid
collision with Sepa Instrument